### PR TITLE
Task/improve modals

### LINKED ISSUE #94: Improve modals
We would like to improve existing modal in various ways. Changes should be done for confirmation modal, wallet editor modal and transaction editor modal

## Buttons
- Make all buttons 50% wide, secondary button should use angular material stroked style, primary button should use angular material raised style
- Make this change for all modals
- Add 30px padding on top of actions to separate them from the content
Track Expenses 2023-02-06 11-52-34
---
## Sizes
- Adjust default global config for modals to have min width 300px and max width 90vw
- Inputs in modals should take all available width, if there is one input in single row in modal it should take 100% width, if there are two inputs in single row those should take 50% with 10px gap between them etc
---
## Forms
- Remove `appearance="fill"` from all inputs and globally adjust config so all inputs have `outline` appearance

### PR DESCRIPTION
**Pull request by Bartexs**
Closes #94 

---

I confirm that (check also those that do not apply to this PR):
- [X] New code is covered with tests and coverage requirements are met
- [X] All tests are passing
- [X] Build is not failing
- [X] Mobile version of new functionality is displayed properly
- [X] All linter issues are resolved
- [X] All conflicts are resolved and branch is rebased on top of the up-to-date parent branch
- [X] All issue requirements are implemented
- [X] At least two reviewers are assigned
- [X] I'm assigned as a assignee
- [X] PR is linked to the related issue

### CODE REVIEW

**Review by mirekgab — DISMISSED** [review]

File: src/assets/styles/components/modals.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    display: flex;
+    padding-top:30px;
+}
+
+div.mat-dialog-actions button, mat-form-field {
+    flex: 1;
+}
+
+mat-dialog-container {
+    min-width: 300px;
```
Comment (purbanski-deftcode) [inline src/assets/styles/components/modals.scss]: You should use global config for this:
https://material.angular.io/components/dialog/overview#specifying-global-configuration-defaults
https://material.angular.io/components/dialog/api#MatDialogConfig

File: src/assets/styles/components/modals.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+div:has( + mat-form-field) {
+    display: flex;
+    gap: 10px;
+    flex-wrap: wrap;
+}
+
+mat-form-field {
+    width: 100%;
+}
+
```
Comment (purbanski-deftcode) [inline src/assets/styles/components/modals.scss]: Please remove 2 out of 3 empty lines in the end of the file

File: src/assets/styles/components/modals.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+div.mat-dialog-actions button, mat-form-field {
+    flex: 1;
+}
+
+mat-dialog-container {
+    min-width: 300px;
+    max-width: 90vw;
+}
+
+div:has( + mat-form-field) {
```
Comment (purbanski-deftcode) [inline src/assets/styles/components/modals.scss]: Remov space before `+`

File: src/assets/styles/components/modals.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
+div.mat-dialog-actions {
+    display: flex;
+    padding-top:30px;
```
Comment (purbanski-deftcode) [inline src/assets/styles/components/modals.scss]: Please add space after `:`

Comment (purbanski-deftcode) [inline src/assets/styles/components/modals.scss]: And my bad, 30px seems to be too much, lets go with 20px :)

**Review by purbanski-deftcode — CHANGES_REQUESTED** [review]

**Review by purbanski-deftcode — APPROVED** [review]

**Review by Iwona007 — APPROVED** [review]

**Pull request merged**